config/webpack.config.js

```diff
@@ -23,6 +24,11 @@ var resolveModules = [  path.join(__dirname, '..', 'webpack'),
                         'node_modules/',
                      ].concat(pluginUtils.pluginNodeModules(plugins));
 
+// Detect supported locales from folders in ./locale/
+const localeDir = path.join(__dirname, '..', 'locale')
+const localeSubdirs = fs.readdirSync(localeDir).filter(f => fs.statSync(path.join(localeDir, f)).isDirectory());
+const supportedLocales = localeSubdirs.map(d => d.split('_')[0]).join('|')
```

Comment:
This produces duplicates for languages that appear twice such as `en` or `zh`

Reply:
Good catch, updated